pkg/analyzer/parser.go

```diff
@@ -60,6 +61,7 @@ func (p *NSXConfigParser) init() {
 	p.servicePathsToObjects = map[string]*collector.Service{}
 	p.groupToVMsListCache = map[*collector.Group][]endpoints.EP{}
 	p.servicePathToConnCache = map[string]*netset.TransportSet{}
+	p.allRuleBlocks = map[string]*endpoints.RuleBlock{}
```

Comment:
add comment about this map, what is the key

Reply:
done